src/actions/google/test_google_drive.ts

```diff
@@ -26,7 +26,16 @@ function expectGoogleDriveMatch(request: Hub.ActionRequest, optionsMatch: any) {
     }))
 
   return chai.expect(action.execute(request)).to.be.fulfilled.then(() => {
-    chai.expect(fileUploadSpy).to.have.been.calledWithMatch(optionsMatch)
+    const gDrivePayload = fileUploadSpy.args[0][0]
```

Comment:
the sinon matcher does not play nice with the stream objects, it seems; instead I just read from the stream below and assert the streamed value is as expected.